### ISSUE TITLE
lit-a11y/list exists in recommended rules list but not in the overview.md docs

### ISSUE THREAD

**Issue by Shadebane** [open-wc/open-wc#2796]
## Expected behavior
['Recommended rules' ](https://github.com/open-wc/open-wc/blob/7a5123dbff4bce7c23b7d18445a450569693e068/packages/eslint-plugin-lit-a11y/lib/index.js#L38) is at a minimum a subset of ['Supported rules'](https://github.com/open-wc/open-wc/blob/7a5123dbff4bce7c23b7d18445a450569693e068/docs/docs/linting/eslint-plugin-lit-a11y/overview.md#supported-rules), so that there is no confusion around which rules might be turned on by default when using the recommended rules

## Actual Behavior
'lit-a11y/list' is missing from the list of [supported rules](https://github.com/open-wc/open-wc/blob/7a5123dbff4bce7c23b7d18445a450569693e068/docs/docs/linting/eslint-plugin-lit-a11y/overview.md#supported-rules), which is causing confusion on whether or not it is a rule that is meant to be in the [recommended list](https://github.com/open-wc/open-wc/blob/7a5123dbff4bce7c23b7d18445a450569693e068/packages/eslint-plugin-lit-a11y/lib/index.js#L38) of linting rules or if it has been deprecated or something. 

## Additional context

Plugin: eslint-plugin-lit-a11y v4.1.2

Just trying to get some clarity on whether or not this rule is meant to be in the supported rules list - it looks like it may just have been missed when this file was originally ported over 4 years ago. I see no reference to any discussion on it the [original PR](https://github.com/open-wc/open-wc/pull/1875/files#diff-e60898a9393e968bb90e68f77018a720309e87a29c9aa20276a00f7f8a8fa5d5R83-R106) that would suggest otherwise, at any rate.

**Comment by thepassle** [comment]
Looks like the docs just got out of sync with source code. Would you be willing to create a pr to update it?

**Comment by Shadebane** [comment]
> Looks like the docs just got out of sync with source code. Would you be willing to create a pr to update it?

Done. I kept the scope to this issue but there seem to be other examples outside this rule( [definition-list](https://github.com/open-wc/open-wc/blob/master/packages/eslint-plugin-lit-a11y/docs/rules/definition-list.md), [heading-hidden](https://github.com/open-wc/open-wc/blob/master/packages/eslint-plugin-lit-a11y/docs/rules/heading-hidden.md), - basically any rule added since the overview list was established).

 I don't mind syncing the overview list but i wonder if a better solution couldn't be found. I don't have any insight into the reason behind the division between the package folder & the docs folder (when the package folder also has its own docs).

**Comment by Shadebane** [comment]
PR has been merged
